README.md

```diff
@@ -104,7 +107,7 @@ _⚠️  The values set has to be correctly typed but are **not** validated._
 
 ## 🔧 Custom validators
 
-By default, `valienv` only exports 3 validators: `str` (for `string`), `nbr` (for `number`) and `bool` (for `boolean`).<br>
+By default, `valienv` only exports 4 validators: `str` (for `string`), `nbr` (for `number`), `bool` (for `boolean`) and `oneOf` (for union of strings).<br>
```

Comment:
Here I'm not sure this is a good end of sentence because as you said this is a helper to create a validator and not really a validator. But from user point of view it looks like a validator.  
And I'm not sure saying `union of strings` is correct (I think yes because it returns a union type)

Reply:
I think this is good enough:

> By default, `valienv` only exports 3 validators: `str` (for `string`), `nbr` (for `number`) and `bool` (for `boolean`) and `oneOf`, a helper to create validators for union of string literals.

- https://mariusschulz.com/blog/string-literal-types-in-typescript
- https://en.wikipedia.org/wiki/String_literal

Reply:
I just updated this sentence, thanks for the links